Xamarin.Essentials/Screenshot/Screenshot.macos.cs

```diff
@@ -17,6 +17,7 @@ static Task<ScreenshotResult> PlatformCaptureAsync()
         {
             var image = ApplicationServices.GetScreenshot();
             var result = new ScreenshotResult(new NSImage(image, new CGSize(image.Width, image.Height)));
+            image?.Dispose();
```

Comment:
Remove this and use 'using var image = '

Reply:
Have to still get used to it... Thanks. It`s commited.